_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -359,9 +360,7 @@ public function show_setup_wizard() {
 		 *
 		 * @param array $jetpack_show_setup_wizard If true, the Setup Wizard will be displayed. Otherwise it will not display.
 		 */
-		return apply_filters( 'jetpack_show_setup_wizard', false )
-			&& Jetpack::is_active()
-			&& current_user_can( 'jetpack_manage_modules' );
+		return Jetpack_Wizard::can_be_displayed();
```

Comment:
Since you're removing the filter here, can you move the docblock to `_inc/lib/class-jetpack-wizard.php`, so the parser can still catch it?

Reply:
I think I did this correctly, let me know if anything needs to change.